de.dlr.sc.virsat.model.extension.funcelectrical/src/de/dlr/sc/virsat/model/extension/funcelectrical/excel/exporter/FuncElecExporter.java

```diff
@@ -42,69 +42,60 @@
 import de.dlr.sc.virsat.model.extension.ps.model.ElementOccurence;
 import de.dlr.sc.virsat.model.extension.ps.model.ElementRealization;
 
-
-
-
 /**
  * Class for exporting excel
- * @author bell_er
- *
  */
 public class FuncElecExporter implements IExport {
 	private final String defaultTemplatePath = "/resources/ExcelExportTemplate.xlsx";
 	private ExcelExportHelper helper;
 
-	public static final String[] EXPORTABLE_SEIS = { 
+	public static final String[] EXPORTABLE_SEIS = {
 			ElementDefinition.class.getSimpleName(),
-			ElementConfiguration.class.getSimpleName(), 
-			InterfaceTypeCollection.class.getSimpleName(), 
+			ElementConfiguration.class.getSimpleName(),
+			InterfaceTypeCollection.class.getSimpleName(),
 			ElementRealization.class.getSimpleName(),
 			ElementOccurence.class.getSimpleName()
 	};
+
 	@Override
 	public void export(EObject eObject, String path, boolean useDefaultTemplate, String templatePath) {
 		StructuralElementInstance parentSc = (StructuralElementInstance) eObject;
-		ArrayList <StructuralElementInstance> allSei = new ArrayList<StructuralElementInstance>();
-		allSei.add(parentSc);
-		allSei.addAll(parentSc.getDeepChildren());
-		for (StructuralElementInstance sc : allSei) {
-			String selectedSEIType = sc.getType().getName();
+		ArrayList <StructuralElementInstance> seiList = new ArrayList<StructuralElementInstance>();
+		seiList.add(parentSc);
+		seiList.addAll(parentSc.getDeepChildren());
+		for (StructuralElementInstance sei : seiList) {
+			String selectedSEIType = sei.getType().getName();
 			if (Stream.of(EXPORTABLE_SEIS).anyMatch(exportable -> exportable.equals(selectedSEIType))) {
-				InputStream is = null;
+				InputStream iStream = null;
 				try {
 					if (useDefaultTemplate) {
-						is = Activator.getResourceContentAsString(defaultTemplatePath);
+						iStream = Activator.getResourceContentAsString(defaultTemplatePath);
 					} else {
-						is = Activator.getResourceContentAsString(templatePath);
+						iStream = Activator.getResourceContentAsString(templatePath);
 					}
-				} catch (IOException e) {
-					Activator.getDefault().getLog().log(new Status(Status.WARNING, Activator.getPluginId(), "Could not locate the template file"));
-				}
-				export(sc, is);
-
-				File file = new File(path + "/" + sc.getFullQualifiedInstanceName() + ".xlsx");
-				try {
+					export(sei, iStream);
+					File file = new File(path + "/" + sei.getFullQualifiedInstanceName() + ".xlsx");
 					FileOutputStream out = new FileOutputStream(file);
 					helper.getWb().write(out);
 				} catch (IOException e) {
-					Status status = new Status(Status.ERROR, Activator.getPluginId(), "Failed to perform an export operation! ", e);
+					Status status = new Status(Status.ERROR, Activator.getPluginId(), "Failed to perform an export operation! \n" + e.getMessage(), e);

```

Comment:
I think we will get the error message printed anyways because we are passing the exception.